src/YHttpWidgetsActionHandler.h

```diff
@@ -31,11 +31,16 @@
 #include "YInputField.h"
 #include "YItem.h"
 #include "YMultiSelectionBox.h"
+#include "YMenuItem.h"
 #include "YSelectionBox.h"
 #include "YTimeField.h"
 #include "YWidgetFinder.h"
 #include "YWidget.h"
 
+#include <boost/algorithm/string.hpp>
+
+#define YUILogComponent "rest-api"
```

Comment:
Missed that one, it breaks compilation of ncurses library, because we cannot redefine log component. I will fix that.